src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1430,8 +1430,8 @@ def add_hostname(cmd, resource_group_name, webapp_name, hostname, slot=None):
     if slot is None:
         return client.web_apps.create_or_update_host_name_binding(resource_group_name, webapp.name, hostname, binding)
 
-    return client.web_apps.create_or_update_host_name_binding_slot(resource_group_name, webapp.name, hostname, binding,
-                                                                   slot)
```

Comment:
lets update this to use positional arguments so that we don't run into this issue again https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#positional-arguments